### PR TITLE
feat(typography): add EMA custom - INNO-875

### PR DESCRIPTION
**Pull request by emeryro**
Add fonts, headings, paragraphs and links

### CODE REVIEW

**Comment by kalinchernev** [comment]
We miss HAL-Patch-INNO making previews ;)

File: framework/base/variables/_colors.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@ $ema-colors: (
     green-1: #06854b,
     green-2: #13b66d,
     green-3: #89cdad,
-    black: #1e1e1e,
+    ema-black: #1e1e1e,
```
Comment (raduchiriac) [inline framework/base/variables/_colors.scss]: Use quotes for all keys or at least for `"black"`

**Review by raduchiriac — CHANGES_REQUESTED** [review]
- [x] rewrite the map keys with wrapping quotes
- [x] line-height are unitless
- [x] we agree we don't need README for now

**Comment by HAL-Patch-INNO** [comment]
Deploy preview ready!

Built with commit 15ed2065f6fe8ae338302453f33d0dfc8f810071

https://deploy-preview-6--ema-styleguide.netlify.com

**Review by raduchiriac — APPROVED** [review]

**Pull request merged**